File: app/src/main/java/com/example/redscate/AlertHomeActivity.java
package com.example.redscate;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class AlertHomeActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_alert_home);

        findViewById(R.id.alertNoButton).setOnClickListener(v -> finish());
        findViewById(R.id.alertYesButton).setOnClickListener(v ->
                startActivity(new Intent(this, ChatActivity.class)));

        findViewById(R.id.profileButton).setOnClickListener(v ->
                startActivity(new Intent(this, ProfileActivity.class)));
        findViewById(R.id.helpButton).setOnClickListener(v -> {
            // Already on the alert screen; avoid stacking the same dialog activity.
        });
        findViewById(R.id.bottomDocButton).setOnClickListener(v ->
                startActivity(new Intent(this, LibraryActivity.class)));
    }
}

File: app/src/main/java/com/example/redscate/AlertHomeActivity.kt
package com.example.redscate

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class AlertHomeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_alert_home)

        findViewById<View>(R.id.alertNoButton).setOnClickListener { finish() }
        findViewById<View>(R.id.alertYesButton).setOnClickListener {
            startActivity(Intent(this, ChatActivity::class.java))
        }

        findViewById<View>(R.id.profileButton).setOnClickListener {
            startActivity(Intent(this, ProfileActivity::class.java))
        }
        findViewById<View>(R.id.helpButton).setOnClickListener {
            // Already on the alert screen; avoid stacking the same dialog activity.
        }
        findViewById<View>(R.id.bottomDocButton).setOnClickListener {
            startActivity(Intent(this, LibraryActivity::class.java))
        }
    }
}
